Move forward rocksdb minimum

argakiig opened the pull request.

Commit:
correct artifact for rocksdb and osx

guilhermelawless reviewed (changes requested):
New version required for #2930

Commit:
rocksdb osx and linux to 6.5.2 for #2930
rocksdb windows to 6.6.4 as that is what vcpgk comes with currently and
is greater than required 6.5.2

Commit:
Merge branch 'develop' into rocksdb_osx

Commit:
force overwrite of existing on windows when extracting incase others
exist already
